src/management-system-v2/components/process-list.tsx

```diff
@@ -87,42 +82,135 @@ const ProcessList: FC<ProcessListProps> = ({
   const lastProcessId = useLastClickedStore((state) => state.processId);
   const setLastProcessId = useLastClickedStore((state) => state.setProcessId);
 
-  const [rerender, setRerender] = useState(false);
+  const { preferences, addPreferences } = useUserPreferences();
 
-  const triggerRerender = () => {
-    /*  Timeout necessary for animation and table resize */
-    setTimeout(() => {
-      setRerender(!rerender);
-    }, 200);
-  };
+  const {
+    'process-list-columns': selectedColumns,
+    'ask-before-deleting-single': openModalWhenDeleteSingle,
+  } = preferences;
 
-  const actionBarGenerator = useCallback((record: Process) => {
-    return (
-      <>
-        <Tooltip placement="top" title={'Preview'}>
-          <EyeOutlined
-            onClick={() => {
-              setPreviewProcess(record);
-              setPreviewerOpen(true);
-            }}
-          />
-        </Tooltip>
-        <Tooltip placement="top" title={'Copy'}>
-          <CopyOutlined />
-        </Tooltip>
-        <Tooltip placement="top" title={'Export'}>
-          <ExportOutlined
-            onClick={() => {
-              onExportProcess([record.definitionId]);
-            }}
-          />
-        </Tooltip>
-        <Tooltip placement="top" title={'Delete'}>
-          <DeleteOutlined />
-        </Tooltip>
-      </>
-    );
-  }, []);
+  const ability = useAuthStore((state) => state.ability);
+
+  const clipAndHighlightText = useCallback(
+    (dataIndexElement, record, index) => {
+      const withoutSearchTerm = dataIndexElement?.split(search);
```

Comment:
Fuse has a `includeMatches` option to get the matched characters

Reply:
I thought about using the returned matches from Fuse for highlighting but I think they do not integrate as nicely into AntDesing's Table-Component. With a simple clip-and-highlight function for strings, that can be passed as the render function for each column, I think we are better of with such a simple function, as of for now the render functions gets invoked anyways and that way we do not have to differentiate between the columns (Fuse's search keys). 

I would - nevertheless - propose to use Fuse's matches once the Table will be replaced with a custom component. 

I made the highlighting case insensitive for now.